### DISCUSSION TITLE
Is there any way to gather info about software that runs on server during associations?

### DISCUSSION THREAD

**Discussion by qarmin** [Enet4/dicom-rs#619] [Q&A] [answered]
I have preliminarily reviewed the available options and did not notice this functionality in dicom-rs or other libraries, but it is possible that I did not examine them closely enough.

I wanted to get info about software that server runs on `OrthanC 4.6.0, dcmtk 3.6.4-2`

**Comment by Enet4** [comment, marked as answer]
Thank you for bringing this up. With your contribution in #622 it should now be possible for client application entities to fetch the implementation details of the server application entity.
